prometheus-to-sd/translator/translator.go

```diff
@@ -143,6 +143,11 @@ func translateOne(config *config.CommonConfig,
 		EndTime: time.Now().UTC().Format(time.RFC3339),
 	}
 	metricKind := extractMetricKind(mType)
+	// Prometheus Cumulative -> Stackdriver Gauge translation
+	if metricKind == "GAUGE" && isMetricGaugeToCumulativeWhitelisted(name, config) {
+		metricKind = "CUMULATIVE"
+		interval.StartTime = start.UTC().Format(time.RFC3339)
+	}
```

Comment:
You need also to modify setValue function, because it's still going to use wrong field.

Reply:
Even though I change metricKind, the Prometheus type does not change. For example, if metricKind is changed from "GAUGE" to "CUMULATIVE", we still want to get the value out of the Prometheus Gauge since the variable "metric" passed into setValue would still be Gauge